src/gordon_gcp/plugins/service/enricher.py

```diff
@@ -195,6 +197,24 @@ def _get_internal_ip(self, instance_data):
                 fqdn, external_ip, default_ttl, msg_logger)
         ]
 
+    async def _get_matching_records(self, instance_resource_url):
+        # Get all Google DNS records for the zone
+        project = self.config.get('project')
+        managed_zone = self.config['managed_zone']
```

Comment:
I might be misunderstanding the purpose of the managed_zone config and how it relates separately to the publisher, enricher, janitor, etc. How many publishers, enrichers, etc. are there configured for each managed zone? Can the enricher have `managed_zone` as part of its config?

Reply:
Moved the `managed_zone` attribute up to the `gcp` level in the config to illustrate where the managed_zone configuration might be.